### PR TITLE
Cleanup - Bugzilla-2957

### PR DESCRIPTION
**Pull request by ondrejtomcik**

### CODE REVIEW

File: swagger2.0/oic.r.cloudopenapi.swagger.json [hunk 1]
```diff
@@ -2,10 +2,13 @@
   "swagger": "2.0",
   "info": {
     "title": "OCF Cloud Open API",
-    "version": "0.0.1"
+    "version": "0.4.0",
```
Comment (marktrayer) [inline swagger2.0/oic.r.cloudopenapi.swagger.json]: What we've done for the version field is simply put in the date of the most recent change.  Pick your own date format.  Trying to do actual version style fields ends up with inconsistent content.

File: swagger2.0/oic.r.cloudopenapi.swagger.json [hunk 2]
```diff
@@ -2,10 +2,13 @@
...
...et:\n        1.  Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.\n        2.  Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.\n\n        THIS SOFTWARE IS PROVIDED BY THE Open Connectivity Foundation, INC. \"AS IS\" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE OR WARRANTIES OF NON-INFRINGEMENT, ARE DISCLAIMED.\n        IN NO EVENT SHALL THE Open Connectivity Foundation, INC. OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)\n        HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.\n"
```
Comment (marktrayer) [inline swagger2.0/oic.r.cloudopenapi.swagger.json]: Copyright should just be 2019.

File: swagger2.0/oic.r.cloudopenapi.swagger.json [hunk 3]
```diff
@@ -24,7 +27,7 @@
     }
   ],
   "paths": {
-    "/devices": {
+    "/api/v1/devices": {
```
Comment (marktrayer) [inline swagger2.0/oic.r.cloudopenapi.swagger.json]: Why are you prepending /api/v1 to all the paths?  Doesn't this force the URI exposed by the Clouds to include this?

**Review by marktrayer — COMMENTED** [review]
See comments embedded.

File: swagger2.0/oic.r.cloudopenapi.swagger.json (hunk 3 above)
Comment (ondrejtomcik) [reply swagger2.0/oic.r.cloudopenapi.swagger.json]: Yes, that's the goal. It was like that also in the specification and also in the swagger, through the basePath parameter. But Dongjoo didn't know about the basePath parameter and others might be also confused, so I added it to each endpoint. It's based on the spec.

Comment (marktrayer) [reply swagger2.0/oic.r.cloudopenapi.swagger.json]: This is different however to what you had before, resolving properly the basePath that you've now deleted you'd have:
/v1/devices
You've changed that to:
/api/v1/devices

I'm very leery of just putting this as an accepted change in the Swagger without having the TG ok this.

**Review by marktrayer — COMMENTED** [review]
Please see comment.

File: swagger2.0/oic.r.cloudopenapi.swagger.json (hunk 3 above)
Comment (ondrejtomcik) [reply swagger2.0/oic.r.cloudopenapi.swagger.json]: In the spec we have`/api/v1`. It was just forgotten here, typo.

**Pull request merged**